test/types/queries/entry-queries.test-d.ts

```diff
@@ -14,6 +14,8 @@ expectAssignable<
   >
 >({
   'metadata.tags.sys.id[all]': mocks.stringArrayValue,
+  'metadata.concepts.sys.id[all]': mocks.stringArrayValue,
+  'metadata.concepts.descendants.id[in]': mocks.stringArrayValue,
```

Comment:
```suggestion
  'metadata.concepts.descendants[in]': mocks.stringArrayValue,
```

Reply:
Great catch @luizfonseca, actually had to change the query type a bit, as `descendants` doesn't exist on `concepts`.